prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+/* stylelint-disable block-no-empty */
+body {
+  font-family: sans-serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+.container {
+  width: 100%;
+  max-width: 992px;
+  margin: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.column {
+  flex-basis: calc(100%/3);
+  padding-right: 8px;
+  padding-left: 8px;
+}
+
+img { width: 100%; }
+  </style>
+</head>
+<body>
+
+<h1>Indiana Jones</h1>
+<div class="container">
+  <div class="row">
+    <div class="column"><img src="images/raiders.jpg" alt="raiders of the lost ark"></div>
+
+        <div class="column"><img src="images/temple.jpg" alt="tempe of doom"></div>
+            <div class="column"><img src="images/crusade.jpg"></div>
```

Comment:
Hi Jay, for this exercise try including the movie posters inside the columns with the other text.